frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
+
+    .commands__cli {
+        height: 50vh;
```

Comment:
Same as above. We cannot assume it's height because we don't know where it will be used. Same aproach, have full width/height of the parent.